jerry-core/include/jerryscript-compiler.h

```diff
@@ -40,6 +40,7 @@ extern "C"
 #define JERRY_ATTR_NORETURN __attribute__((noreturn))
 #define JERRY_ATTR_PURE __attribute__((pure))
 #define JERRY_ATTR_WARN_UNUSED_RESULT __attribute__((warn_unused_result))
+#define JERRY_ATTR_WEAK __attribute__((weak))
```

Comment:
What happens on a compiler other than gcc? The other attributes can be ignored, but this seems mandatory.

Reply:
I reverted the weak attributes for now.